pkg/network/setup/netpod/netpod.go

```diff
@@ -286,6 +295,27 @@ func (n NetPod) composeDesiredSpec(currentStatus *nmstate.Status) (*nmstate.Spec
 			}
 		case iface.SRIOV != nil:
 		case iface.Binding != nil:
+			bindingPlugin, exists := n.bindingPluginsByName[iface.Binding.Name]
```

Comment:
It is a pre-defined attachment in the core which a plugin may use.
Therefore, it may be used with hotplug, but I also do not understand what simplification you are referring to.

Reply:
> It is a pre-defined attachment in the core which a plugin may use.
Therefore, it may be used with hotplug, but I also do not understand what simplification you are referring to.

The support for hotplug/unplug is limited to SR-IOV and bridge binding, if we can avoid having the new binding support hotplug/unplug at least for the first phase, it seems to be easier to implement and maintain.
So I am asking if we can simplify this cod by not supporting hotplug/unplug similar to masquerade dining example (masquerade case in this switch clause)?

Reply:
> Regarding hotplug, I do not recall supporting hotplug for binding-plugins, could you please refer me to some doc/code that support that?

Please refer me to a doc/code that does not support that.

Reply:
https://kubevirt.io/user-guide/network/hotplug_interfaces/#hotplug-network-interfaces